mods/gen2/moves.js

```diff
@@ -12,13 +12,17 @@ exports.BattleMovedex = {
 			if (target.boosts.atk >= 6) {
 				return false;
 			}
+
+			var boost = 0;
+
 			if (target.hp <= target.maxhp / 2) {
-				this.boost({atk: 2});
-				return false;
+				boost = (target.boost ? target.boost.atk : 0) + 2;
```

Comment:
Although seriously now, I think `this.boost({atk: 12})` might be more accurate to how the game works? Not sure, though. @Marty-D?